test/suite/test_import11.py

```diff
@@ -112,6 +113,18 @@ class test_import11(test_import_base):
     tiered_storage_sources = gen_tiered_storage_sources()
     scenarios = make_scenarios(tiered_storage_sources)
 
+    def conn_config(self):
+        return self.tiered_conn_config() + ',statistics=(all)'
+
+    def get_stat(self, stat, uri):
```

Comment:
Do we call this method with `uri` anywhere? If not, we should remove the dead code branch.